_drafts/submitting-your-first-pull-request.markdown

```diff
@@ -0,0 +1,205 @@
+This creates a branch that you own and have control over. You will be making changes in your fork, then submitting a pull request to get your changes merged in to the original repo.
+
+_image_
+
+Notice on my fork’s master branch where it says _This branch is even with HTBox:master_. _HTBox:master_ is the master branch from the original repo and is the upstream for my master branch. When GitHub tells me my branch is even with master that means no changes have happened to HTBox:master and no changes have happened to my master branch. Both branches are identical at this point.
+
+## Cloning your fork to your local machine
+
+Next up, you will want to clone the repo to your local machine. Launch GitHub for Windows and sign in with your GitHub account if you have not already done so. Click on the + icon in the top right, select Clone and select the repo that you just forked. Click the big checkmark button at the bottom and select a location to clone the repo on your local disk.
+
+image
+
+## Create a local branch to do your work in
+
+You could do all your work in your master branch, but this might be a problem if you intend to submit more than one pull request to the project. You will have trouble working on your second pull request until after your first pull request has been accepted. Instead it is best practice to create a new branch for each pull request you intend to submit. 
+
+> As a side note, it is also considered best practice to submit pull requests that solve 1 issue at a time. Don’t fix 10 separate issues and submit a single pull request that contains all those fixes. That makes it difficult for the project owners to review your submission.
+
+We could use GitHub for Windows to create the branch, but we’re going to drop down to the command line here instead. Using the command line to do git operations will give you a better appreciation for what is happening.
+
+To launch the git command line, select your fork in GitHub for Windows, click on the the Settings menu in the top right and select Open in Git Shell.
+
+image
+
+This will open a posh-git shell. From here, type the command `git checkout -b MyNewBranch`, where MyNewBranch is a descriptive name for your new branch.
+
+ image
+
+This command will create a new branch with the specified name and switch you to that branch. Notice how posh-git gives you a nice indication of what branch you are currently working on. 
+
+_Advanced Learning_: Learn more about git branching with this interactive tutorial [http://pcottle.github.io/learnGitBranching](http://pcottle.github.io/learnGitBranching/)
+
+_Pro tip_: posh-git has auto complete. Type git ch + tab will autocomplete to git checkout. Press tab multiple times to cycle through available options. This is a great learning tool!
+
+## Committing and publishing your changes
+Next, you will make your changes just like you normally would. When you are done making your changes, you can a list of your changes by typing the `git status` command.
```

Comment:
When you are done making your changes, you can **see** a list of your changes...